src/Package/Impl/ProjectSystem/Commands/SourceFilesCommand.cs

```diff
@@ -73,7 +73,7 @@ internal sealed class SourceFilesCommand : SendFileCommandBase, IAsyncCommandGro
                         string projectName = properties.GetProjectName();
                         string remotePath = await properties.GetRemoteProjectPathAsync();
 
-                        await SendToRemoteAsync(files, projectDir, projectName, remotePath, CancellationToken.None);
+                        await SendToRemoteAsync(files, projectDir, projectName, remotePath);
```

Comment:
Since command execution doesn't actually provide cancellation token, I'd test

- operation can be canceled via dialog
- can VS be closed while transfer is in progress and if yes, what happens
- what happens if network disconnects during transfer